wgpu-types/src/lib.rs

```diff
@@ -1136,6 +1160,11 @@ pub struct Limits {
     /// The maximum value for each dimension of a `ComputePass::dispatch(x, y, z)` operation.
     /// Defaults to 65535. Higher is "better".
     pub max_compute_workgroups_per_dimension: u32,
+
+    /// Minimal number of workitems in a subgroup. Lower is "better".
```

Comment:
This is backwards from the limit comparison below. I think higher is better, and lower for max, as a narrower range of possibilities is easier for the programmer.

Also this should probably be "invocations" to match the spec, I don't see evidence of "workitems" being used elsewhere.